src/machine/board_k210.go

```diff
@@ -0,0 +1,317 @@
+// +build maixbit
+
```

Comment:
Maybe you can add a link to the datasheet here?
For example https://github.com/kendryte/kendryte-doc-datasheet/blob/master/en/SUMMARY.md. I couldn't easily find a PDF that looked authoritative (although a PDF might be preferred).

Reply:
done